src/pages/cli/graphql/examples-and-solutions.mdx

```diff
@@ -564,4 +564,81 @@ query repsByPeriodAndTotal {
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Advanced
+
+### Batch Put Custom Resolver
+
+Sometimes you need to create objects in bulk, rather than creating individual objects sequentially and waiting for all the requests to complete.
+
+1. Define your schema with a custom mutation. The custom mutation should not be deployed to AppSync beforehand if following these steps, the CLI will attach its own resolver preventing you from attaching a custom resource this way.
+```graphql
+type Todo @model {
+  id: ID!
+  name: String!
+  description: String
+}
+
+type Mutation {
+  batchCreateTodo(todos: [BatchCreateTodo]): [Todo]
+}
+
+input BatchCreateTodo {
+  id: ID
+  name: String!
+  description: String
+}
+```
+
+2. [Create a custom resource for your resolver](/cli/graphql/custom-business-logic/#vtl-resolver) and use the following code snippets as a guide to get started
```

Comment:
For this example, I'd recommend to just add the steps in here inline and then link out to the details a sub point.

this will also allow you to collapse steps 3 - 6 into one code block where you can highlight the specific areas of attention with comments.